data/darktableconfig.xml.in

```diff
@@ -103,30 +103,6 @@
     <longdescription></longdescription>
   </dtconfig>
   <dtconfig>
-    <name>storage/piwigo/overwrite</name>
```

Comment:
isn't this some merging issue? touching piwigo here seems suspicious :)

Reply:
oops :-)